apps/community/src/components/page-header.css.ts

```diff
@@ -1,6 +1,13 @@
 import { style } from '@vanilla-extract/css';
 
+const pageHeaderWrapper = style({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: 12,
+});
+
 const title = style({
+  margin: 0,
   fontSize: '2rem',
   fontWeight: 'bold',
```

Comment:
```suggestion
  fontWeight: 700,
```

`bold`의 경우 700, `semibold`는 600인데요, 아직 CSS에서는 `semibold`를 지정할 수 없어 일관되게 숫자로 표현하는게 좋을 거 같아요